Resources/public/css/styles.css

```diff
@@ -496,3 +496,15 @@ div.sonata-filters-box div.form-group span.input-group-addon {
     padding: 3px 10px;
     font-size: 13px;
 }
+
+.hide {
+    display: none;
```

Comment:
@core23 @OskarStark Hi, can i ask you, what sense of sonata_block.templates.block_base template?